CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,19 @@
 
 import Foundation
 
-print("Hello, World!")
+func makeIceCream () {
```

Comment:
애플의 공식 문서 중 하나인 [Functions](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/functions/#Functions-Without-Parameters)을 보시면 함수 괄호 부분을 붙이는 걸 확인하실 수 있어요!
```suggestion
func makeIceCream() {
```

참고하셔서 아래 함수도 수정 부탁드려요~